Library/Homebrew/test/pathname_spec.rb

```diff
@@ -0,0 +1,302 @@
+require "tmpdir"
+require "extend/pathname"
+require "install_renamed"
+
+describe Pathname do
+  let(:src) { Pathname.new(Dir.mktmpdir) }
+  let(:dst) { Pathname.new(Dir.mktmpdir) }
+  let(:file) { src/"foo" }
+  let(:dir) { src/"bar" }
+
+  after(:each) do
+    FileUtils.rm_rf [src, dst]
+  end
+
+  describe DiskUsageExtension do
+    before(:each) do
+      FileUtils.mkdir_p dir/"a-directory"
+      FileUtils.touch [dir/".DS_Store", dir/"a-file"]
+      File.truncate(dir/"a-file", 1_048_576)
+      FileUtils.ln_s dir/"a-file", dir/"a-symlink"
+      FileUtils.ln dir/"a-file", dir/"a-hardlink"
+    end
+
+    describe "#file_count" do
+      it "returns the amount of files in a directory" do
```

Comment:
Something about this reads a little strangely to me.

<ins>number</ins> of files, maybe?